samples/grid-lite/demo/parse-data/demo.css

```diff
@@ -0,0 +1,24 @@
+@import url("https://cdn.jsdelivr.net/npm/@highcharts/grid-lite/css/grid.css");
+
+#container {
+    padding: 8px;
+}
+
+.highcharts-description {
+    padding: 0 20px;
+}
+
+.hcg-table td[data-column-id="HealthIndicator"] {
+    text-align: center;
```

Comment:
It looks a bit off when it's centred 🤔

Reply:
Currently we have whe width option, so will adjust it.